common/annotations/en.xml

```diff
@@ -1371,364 +1370,363 @@ annotations.
-		<annotation cp="💂‍♀">buckingham palace | guard | helmet | london | woman</annotation>
+		<annotation cp="💂‍♀">buckingham | guard | helmet | london | palace | woman</annotation>
 		<annotation cp="💂‍♀" type="tts">woman guard</annotation>
 		<annotation cp="🥷">assassin | fight | fighter | hidden | ninja | person | secret | skills | sly | soldier | stealth | war</annotation>
 		<annotation cp="🥷" type="tts">ninja</annotation>
-		<annotation cp="👷">build | construction | fix | hardhat | hat | helmet | man | person | rebuild | remodel | repair | work | worker</annotation>
+		<annotation cp="👷">build | construction | fix | hardhat | hat | man | person | rebuild | remodel | repair | work | worker</annotation>
 		<annotation cp="👷" type="tts">construction worker</annotation>
-		<annotation cp="👷‍♂">construction | man | worker</annotation>
+		<annotation cp="👷‍♂">build | construction | fix | hardhat | hat | man | rebuild | remodel | repair | work | worker</annotation>
 		<annotation cp="👷‍♂" type="tts">man construction worker</annotation>
-		<annotation cp="👷‍♀">construction | hardhat | hat | helmet | woman | work | worker</annotation>
+		<annotation cp="👷‍♀">build | construction | fix | hardhat | hat | man | rebuild | remodel | repair | woman | work | worker</annotation>
 		<annotation cp="👷‍♀" type="tts">woman construction worker</annotation>
-		<annotation cp="🫅">crown | monarch | noble | person with crown | regal | royal | royalty</annotation> <!-- 1FAC5 -->
+		<annotation cp="🫅">crown | monarch | noble | person | regal | royal | royalty</annotation> <!-- 1FAC5 -->
 		<annotation cp="🫅" type="tts">person with crown</annotation>
-		<annotation cp="🤴">prince | royal | royalty</annotation>
+		<annotation cp="🤴">crown | fairy | fairytale | fantasy | king | prince | royal | royalty | tale</annotation>
 		<annotation cp="🤴" type="tts">prince</annotation>
-		<annotation cp="👸">crown | fairy tale | fairytale | fantasy | princess | queen</annotation>
+		<annotation cp="👸">crown | fairy | fairytale | fantasy | princess | queen | royal | royalty | tale</annotation>
 		<annotation cp="👸" type="tts">princess</annotation>
-		<annotation cp="👳">person wearing turban | turban</annotation>
+		<annotation cp="👳">person | turban | wearing</annotation>
 		<annotation cp="👳" type="tts">person wearing turban</annotation>
-		<annotation cp="👳‍♂">man | man wearing turban | turban</annotation>
+		<annotation cp="👳‍♂">man | turban | wearing</annotation>
 		<annotation cp="👳‍♂" type="tts">man wearing turban</annotation>
-		<annotation cp="👳‍♀">turban | woman | woman wearing turban</annotation>
+		<annotation cp="👳‍♀">turban | wearing | woman</annotation>
 		<annotation cp="👳‍♀" type="tts">woman wearing turban</annotation>
-		<annotation cp="👲">cap | gua pi mao | guapimao | hat | person | person with Chinese cap | person with skullcap | skullcap</annotation>
+		<annotation cp="👲">cap | Chinese | gua | guapimao | hat | mao | person | pi | skullcap</annotation>
```

Comment:
Fixed in spreadsheet to '| cap | Chinese | gua pi mao | guapimao | hat | skullcap |'

Reply:
It's been disassociated from the spreadsheet, so if you can let me know which items you changed, I can adjust the PR.

Reply:
"👲" | cap | Chinese | gua pi mao | guapimao | hat | skullcap |
"🖤" | black | evil | heart | wicked | 143
"🧡" | heart | orange | 143
"🤎" | brown | heart | 143
"🤍" | heart | white | 143
"🏻" | type 1-2 | light | skin tone |
"🏼" | type 3 | medium-light | skin tone | 
"🏽" | type 4 | medium | skin tone | 
"🏾" | type 5 | medium-dark | skin tone |
"🏿" | type 6 | dark | skin tone |
"👑" | clothing | crown | king | medieval | queen | royal family | royalty | win |
"😊" | blush | face | glad | satisfied | smile | smiling eyes |
"🥰" adore | crush | face | hearts | I love you | ily | love | romance | smile | smiling |